sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -2067,6 +2082,10 @@ object DatePart {
        224
       > SELECT _FUNC_('SECONDS', timestamp'2019-10-01 00:00:01.000001');
```

Comment:
Nit: is this missing parens, or space, after `timestamp` or am I missing something?

Reply:
This code is executed by the https://github.com/apache/spark/blob/7aca0dd658b8bda05574b3df3254aaf66eb2a174/sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala#L149-L197 , and this is the syntax of timestamp literals.